econet_base.yaml

```diff
@@ -9,6 +9,7 @@ substitutions:
   board: esp32dev
   github_ref: main
   external_components_source: github://esphome-econet/esphome-econet@${github_ref}
+  econet_alarm_update_interval: 30s
```

Comment:
The name says "alarm" but it looks like this is also used for non-alarm config components, e.g. "Screen Adjustment Lock."

Is 30s slow enough to reduce bus traffic? Seems like 60s is fast enough for these sensors.

Or do config entries need to be on the request_mod with the alarms? Not sure how/why these things were allocated to different request_mod setups.

Reply:
I think 30s is slow enough, but I would have no concern dropping it to 60s either. My main intent with this is actually to be able to speed up the more changing/"interesting" datapoints. I'll update this to 60s for the time being. If anyone has a preference to stay with 30s (as the default) speak up now! Anyone will be able to override themselves too.